api/src/main/java/org/accula/api/config/SecurityConfig.java

```diff
@@ -20,8 +21,9 @@
      * @return custom filter chain
      */
     @Bean
+    @NotNull
```

Comment:
Redundant annotation. A bean itself by default is `Scope.SINGLETON` thus it can't be `null`.

Reply:
fixed